invenio_files_rest/config.py

```diff
@@ -135,3 +141,17 @@
 
 FILES_REST_XSENDFILE_RESPONSE_FUNC = create_file_streaming_redirect_response
 """Function for the creation of a file streaming redirect response."""
+
+FILES_REST_FORCE_DELETE_ORPHAN_FILE = never_force_delete
```

Comment:
Super minor:
```suggestion
FILES_REST_FORCE_DELETE_ORPHAN_FUNC = never_force_delete
```

Reply:
An alternative to this new configuration variable altogether would be to have the shared task accept an argument:
```
@shared_task(ignore_result=True)
def clear_orphaned_files(force_delete_orphan_func):
    # ...
```

You could provide the argument (boolean value or function) via the configuration of scheduled celery tasks (`CELERY_BEAT_SCHEDULE`).
Something like:
```
    'clear-orphan-files': {
        'task': 'invenio_files_rest.tasks.clear_orphaned_files',
        'schedule': 60 * 60 * 24,
        'args': (True,)
    },
```

WDYT?